ember-toucan-core/src/components/form/fieldset.ts

```diff
@@ -4,10 +4,29 @@ import { assert } from '@ember/debug';
 interface ToucanFormFieldsetComponentSignature {
   Element: HTMLFieldSetElement;
   Args: {
+    /**
+     * Provide a string to this argument to render an error message and apply error styling to the Field.
+     */
     error?: string;
+
+    /**
+     * Sets the fieldset to an errored-state via styling.
+     */
     hasError?: boolean;
```

Comment:
Curious why there is this separate `hasError` arg, when there is already an `error` arg above.  How do they interact with each other?

Reply:
I was wondering the same - looks like we aren't using it anymore!  I'll delete it.  Thanks a bunch!